_team/mattwang.md

```diff
@@ -0,0 +1,17 @@
+---
+firstname: "Matthew"
+lastname: "Wang"
+title: "President"
+secondary: "Web Dev Lead"
+group: "board"
+img: "mwang.jpg"
+email: "matt@matthewwang.me"
+github: "malsf21"
+year: 2022
+
+positions:
+  - year: 2019-2020
+    title: Dev Team Director
+---
+
+Matt's page
```

Comment:
Do you mind deleting this for now? I'll get back to it later, I promise!

Reply:
Another artifact from messing around. Nice catch.